larpandora/LArPandoraEventBuilding/LArPandoraShower/Tools/Cheating/ShowerTrackFinderCheater_tool.cc

```diff
@@ -122,22 +127,37 @@ namespace ShowerRecoTools {
     ShowerEleHolder.GetElement(fShowerDirectionInputTag,ShowerDirection);
 
     auto const hitHandle = Event.getValidHandle<std::vector<recob::Hit> >(fHitModuleLabel);
-    std::vector<art::Ptr<recob::Hit> > hits;
-    art::fill_ptr_vector(hits, hitHandle);
 
     // Get the hits associated with the space points
     art::FindManyP<recob::SpacePoint> fmsph(hitHandle, Event, fPFParticleLabel);
     if(!fmsph.isValid()){
       throw cet::exception("ShowerTrackFinderCheater") << "Spacepoint and hit association not valid. Stopping.";
     }
 
+    std::vector<int> trueParticleIdVec;
+
+    // If we have an electron, take the hits from the primary only
+    // This will also cover the cases Pandora misclassify a track as a shower
+    if (trueParticle->PdgCode() != 22)
+    {
+      trueParticleIdVec.push_back(trueParticle->TrackId());
+    } else {
```

Comment:
Sandwiching the else between the braces makes this really hard to read.  Can you move the else to the next line?

Reply:
I spoke to @AndyChappell offline and we agreed that I would make a second PR once this one is merged to clang-format all of the PandoraShower code. Unless you feel particularly strongly I will leave this one until then?

Reply:
Fine by me 👍